pkg/testutil/setup.go

```diff
@@ -39,9 +38,9 @@ func SetupTestingEnv(params *SetupTestingEnvParams) (logging.Logger, api.ClientW
 	viper.SetDefault("setup_lakefs_timeout", defaultSetupTimeout)
 	viper.SetDefault("endpoint_url", "http://localhost:8000")
 	viper.SetDefault("s3_endpoint", "s3.local.lakefs.io:8000")
-	viper.SetDefault("access_key_id", "")
-	viper.SetDefault("secret_access_key", "")
-	viper.SetDefault("storage_namespace", fmt.Sprintf("s3://%s/%s", params.StorageNS, xid.New().String()))
+	viper.SetDefault("access_key_id", params.AdminAccessKeyID)
+	viper.SetDefault("secret_access_key", params.AdminSecretAccessKey)
+	viper.SetDefault("storage_namespace", fmt.Sprintf("s3://%s", params.StorageNS))
```

Comment:
I'm not sure about this one - and I'll explain:
Setting default means: if you don't set something this is the default. If you like to say I want a configuration to be something - we need to call Set.
The one that running the tests can control the access key id/secret as we set the default. But in this case as far as I understand we don't want to say this is the default/fallback - these are the parameters we like to use.
So when we decided to use these values we should have just call viper.Set instead of passing 'params'.